src/components/buttons/tertiary.tsx

```diff
@@ -4,19 +4,22 @@ export interface TertiaryButtonProps {
 	label: string | React.ReactNode;
 	onClick: () => void;
 	disabled?: boolean;
+	type?: "button" | "submit";
 }
 
 export const TertiaryButton: React.FC<TertiaryButtonProps> = ({
 	label,
 	onClick,
 	disabled,
+	type,
 }) => {
 	return (
 		<button
 			className={`hover:text-bg-gdk-light-blue p-y-3.5 flex
       h-[51px] w-fit items-center justify-center rounded-[10px] font-semibold text-gdk-blue enabled:hover:text-gdk-light-blue disabled:text-gdk-light-gray`}
 			disabled={disabled}
 			onClick={onClick}
+			type={type ?? "button"}
```

Comment:
just for consistency, in the PrimaryButton, we used a default value for the type in the component arguments instead of the html button attributes
```js
export const PrimaryButton: React.FC<PrimaryButtonProps> = ({
	label,
	onClick,
	disabled,
	type = "button",
}) 
```